### DISCUSSION TITLE
Please extend Privacy Sandbox tokens validity

### DISCUSSION THREAD

**Discussion by lbdvt** [privacysandbox/privacy-sandbox-dev-support#50] [General] [answered]
Criteo has currently a set of four tokens (on criteo.com and criteo.net domains) for the "Privacy Sandbox Relevance and Measurement".

While the OT has been extended to Nov 30, new tokens have been provided and current ones expire on **August 24th**. This is too short to deploy these new tokens in all required apps, and we risk not being able to use the Privacy Sandbox API for several days.

As Topics, FLEDGE, and Attribution Reporting have a 30 days "init phase", this inactivity period may impact the collected data and our tests for a month.

While we agree to use the new tokens in the future, could you please extend the life of the current ones by two weeks, for us to have the time to do the switch?

Thanks

**Comment by lukwlodarczyk** [comment]
At RTB House, we meet the very same issue. 

The announced deadline (August 24th) is too short for deploying changes.
We want to highlight that some proposals tested in OT require collaboration between multiple parties registering owned domains. This dependence might bring additional complexity to the process, impacting participation and measurement of tests.

Please consider a "grace period" to continue using the features of existing tokens. This would allow any OT participant to have more flexibility in testing and to navigate in the interim period.

**Comment by samdutton** [comment, marked as answer]
For anyone new to this discussion — the [Privacy Sandbox origin trial](https://developer.chrome.com/origintrials/#/view_trial/771241436187197441) has now been renewed.

This means that, if you haven't done so already, you will need to obtain and deploy a new token to access the trial: 

1. Go to your [My Registrations](http://developer.chrome.com/origintrials/#/trials/my) page. 
2. Click the RENEW button for an origin. 
3. Copy the new origin trial token. 
4. Deploy the token. 

Screencast: [goo.gle/ot-renew](https://goo.gle/ot-renew) 

The expiry date for an origin trial is encoded in each token for the trial, and cannot be changed. (You can see examples of this at [ot-decode.glitch.me](https://ot-decode.glitch.me).) Renewed tokens for this trial expire on November 29, 2022.

If the RENEW button is not available or visible, you may need to provide feedback first. This won't take long, and you can always provide more feedback later. The option to renew will be available as soon as feedback is submitted. 

You can provide more than one trial token on a page; Chrome will ignore invalid or expired tokens.
